sql/log_event_server.cc

```diff
@@ -8881,31 +8859,20 @@ Update_rows_log_event::do_after_row_operations(const Slave_reporting_capability
   return error;
 }
 
+void Update_rows_log_event::make_thd_info(thd_info_cache cache)
+{
+  Rows_log_event::make_thd_info(cache, "Update_rows_log_event",
+    {"find_row", "unpack_current_row", "ha_update_row"});
+}
+
 int
-Update_rows_log_event::do_exec_row(rpl_group_info *rgi)
+Update_rows_log_event::do_exec_row(rpl_group_info *rgi, thd_info_cache cache)
 {
-  const bool invoke_triggers= (m_table->triggers && do_invoke_trigger());
-  const char *tmp= thd->get_proc_info();
-  DBUG_ASSERT(m_table != NULL);
```

Comment:
If `m_table` _ever_ becomes `NULL`, `m_table->triggers` would’ve already dereferenced null before the assertion catches it. (x